test/npm-support.ts

```diff
@@ -185,6 +186,27 @@ describe("Npm support tests", () => {
 		assert.isTrue(fs.exists(bplistCreatorFolderPath).wait());
 		assert.isTrue(fs.exists(bplistParserFolderPath).wait());
 	});
+	it("Ensures that scoped dependencies are prepared correctly", () => {
+		// Setup
+		let scopedName = "@reactivex/rxjs";
+		let scopedModule = path.join(projectFolder, "node_modules", "@reactivex/rxjs");
+		let scopedPackageJson = path.join(scopedModule, "package.json");
+		addDependencies(testInjector, projectFolder, {scopedName:  "0.0.0-prealpha.3"}).wait();
+		//create module dir, and add a package.json
+		shelljs.mkdir('-p', scopedModule);
```

Comment:
Why do you need to create this folder and add `package.json`? This should be handled by `preparePlatform`.